python_modules/dagster/dagster/_core/storage/event_log/sqlite/sqlite_event_log.py

```diff
@@ -262,10 +262,11 @@ def get_event_records(
         is_asset_query = event_records_filter and (
             event_records_filter.event_type == DagsterEventType.ASSET_MATERIALIZATION
             or event_records_filter.event_type == DagsterEventType.ASSET_OBSERVATION
+            or event_records_filter.event_type == DagsterEventType.ASSET_MATERIALIZATION_PLANNED
```

Comment:
yikes... maybe we should have some sort of constant for the set of asset events?

and then we can just replace these types of things with:
```event_records_filter.event_type in ASSET_EVENT_TYPES```

Reply:
good call